tests/main/boot-state/task.yaml

```diff
@@ -1,5 +1,10 @@
 summary: smoke test for the boot-state tool
 
+# ARM devices are not supported on ubuntu-core-18/20 due to fw_printenv/setenv are
```

Comment:
perhaps we should eventually try to support doing this kind of test not with `fw_printenv`/`setenv` and friends but instead through `snap debug boot-state ...` much like we do with `snap debug boot-vars`. If that's the case I would suggest we leave a TODO: to that effect in this file or in boot-state to suggest that refactoring (I didn't see anything about that refactoring on master).

Reply:
done